### PR TITLE
Devnet configs updated

### PR DESCRIPTION
**Pull request by mxmar**
These configs are based on `mainnet` files. `genesis.json` has `timestamp` calculated like `MIN_GENESIS_TIME+GENESIS_DELAY` and there is standard eth2 deposit contract allocated.

Only `genesis.ssz` is missing, because timestamps and genesis validators will change.

Documented: https://www.notion.so/lukso/Network-start-40859ce38ab24966bfc1fd226f2e9e05?showMoveTo=true

### CODE REVIEW

**Comment by mxmar** [comment]
I converted to draft, because it needs `genesis.ssz` when we know timestamps (infra and CI/CD for 3030 are ready) and genesis validators count and keys, and generate SSZ at the end. (To generate valid SSZ we need keys and configs with timestamps set).

**Review by frozeman — DISMISSED** [review]

File: devnets/1620/shared/config.yaml [hunk 1]
```diff
@@ -9,10 +9,11 @@ CONFIG_NAME: 'lukso'
...
 # ---------------------------------------------------------------
-# `2**14` (= 16,384)
-MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 110
-# Friday, 14 April 2023 13:45:00 GMT+02:00
-MIN_GENESIS_TIME: 1681731000
+# Min Genesis Validators in Genesis state
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 3000
+# Date and time (GMT): Tuesday, 9 May 2023 20:00:00
+# Shanghai time = 16128 (42 epochs in seconds) + 1683662400 + 2400 - 2
+MIN_GENESIS_TIME: 1683662400
```
Comment (kalote) [inline devnets/1620/shared/config.yaml:16]: this is in the past. Set it up to today 1:40pm UTC => 1683898800 (+40min delay = 2:20pm UTC / 4:20pm berlin time)

File: devnets/1620/shared/genesis.json [hunk 2]
```diff
@@ -0,0 +1,842 @@
...
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "mergeForkBlock": 0,
+    "terminalTotalDifficulty": 0,
+    "shanghaiTime": 1683680926
```
Comment (kalote) [inline devnets/1620/shared/genesis.json:17]: should be 1683917326 to match with my previous comment

File: devnets/1620/shared/genesis.json [hunk 3]
```diff
@@ -0,0 +1,842 @@
...
+  },
+  "number": "0x0",
+  "nonce": "0x1",
+  "difficulty": "0x1",
+  "gasLimit": "0x280de80",
+  "gasUsed": "0x0",
+  "timestamp": "0x645AAFA0",
```
Comment (kalote) [inline devnets/1620/shared/genesis.json:24]: 0x645AA640

File: devnets/1620/shared/genesis.json (hunk 2 above)
Comment (mxmar) [reply devnets/1620/shared/genesis.json]: Updated.

File: devnets/1620/shared/genesis.json (hunk 3 above)
Comment (mxmar) [reply devnets/1620/shared/genesis.json]: 1683898800+2400 is `0x645E4B10`. Updated.

**Review by kalote — APPROVED** [review]

**Review by kalote — APPROVED** [review]

**Pull request merged**